specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/deviceSecurityGroups.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "Security Center",
     "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
-    "version": "2017-08-01-preview"
+    "version": "2019-08-01"
```

Comment:
Please move this change to a new folder 2019-08-01

Reply:
Hi @NullMDR 
This PR actually does not change the API contract and doesn't contain breaking changes.
The same changes were already made to the `stable` version a few months back (see discussion in PR: https://github.com/Azure/azure-rest-api-specs/pull/8820), since the original swagger started to break the SDK generation (the `preview` version remained untouched in that PR).

Now, the `preview` version started to break other, unrelated PRs (see: https://github.com/Azure/azure-rest-api-specs/pull/10418#issuecomment-677697471) (ones that add new resource types). So this PR fixes the `preview` version the same way the `stable` version was fixed (again, no actual API changes are made).

Thanks for reviewing 🙂

Reply:
But you should not update this version field. This field must be align with the upper folder name.

Reply:
@NullMDR just realized what you actually meant... 😬
The API version was changed by mistake. I've reverted that change (my other general comments still stand though 😅)